### PR TITLE
Fix several crashing issues

### PR DESCRIPTION
**Pull request by Strencher**
Fixed the canary startup issue where it said  `process.electronBinding is not a function` and with that, i also fixed a stream crashing issue were the client crashed after discord send corrupt data packages.

### CODE REVIEW

**Review by ItsdMeSoFYou — COMMENTED** [review]
works fine

File: injector/src/index.js [hunk 1]
```diff
@@ -9,8 +9,9 @@ import CSP from "./modules/csp";
 if (!process.argv.includes("--vanilla")) {
     process.env.NODE_OPTIONS = "--no-force-async-hooks-checks";
     app.commandLine.appendSwitch("no-force-async-hooks-checks");
-    process.electronBinding("command_line").appendSwitch("no-force-async-hooks-checks");
-
+    ipcMain.on("register_preload", (_, preload) => {
```
Comment (rauenzi) [inline injector/src/index.js:12]: ipc calls should go in the ipc module and the ipc event string should be defined in the ipc constants

**Comment by rauenzi** [comment]
I can accept and make the change if you like

**Comment by Strencher** [comment]
I can change it, no problem.

**Pull request merged**